lib/mutant/cli.rb

```diff
@@ -153,14 +164,14 @@ def add_debug_options(opts)
       end
       opts.on('--version', 'Print mutants version') do
         puts("mutant-#{VERSION}")
-        Kernel.exit(EXIT_SUCCESS)
+        config.kernel.exit(EXIT_SUCCESS)
```

Comment:
Isn't the default status code for `Kernel.exit` success? If so, then the `EXIT_SUCCESS` argument isn't required.